python/data_extraction/export_data.py

```diff
@@ -40,6 +51,16 @@ def content():
     return filter(lambda link: link, map(slicer, content_generator))
 
 
+def untagged_content():
+    pool = Pool(10)
```

Comment:
My knowledge of multithreaded programming is limited, but how did you decide on 10 processes for this? I'm assuming it should depend on the underlying hardware that will be running this.